tools/cloud-build/provision/daily-tests.tf

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "google_cloudbuild_trigger" "daily_test" {
+  for_each    = data.external.list_tests_py.result
+  name        = "DAILY-test-${each.key}"
+  description = "Runs the '${each.key}' integration test against `develop`"
```

Comment:
why not parametrize "develop" right away? you will also need to run these against release candidate, right?

Reply:
alternatively, you can duplicate this file, with the two lines difference (replacing develop for release-candidate).

Reply:
There is no clarity about release tests yet, either options is fine, but for now I'm keeping it simple.